nilearn/datasets.py

```diff
@@ -26,6 +25,17 @@
 
 import nibabel
 
+# Define _get_time, which is used module-wide to get the current time.
+try:
+    # Use uptime (when available) to avoid reporting time while computer slept.
+    import uptime
+    if uptime.uptime() is None:
+        raise Exception("uptime is not supported by this OS.")
+    _get_time = uptime.uptime
```

Comment:
> I am just wondering if we should bother about uptime. 

I was thinking the same. It seems that the cost / benefit ratio isn't
really very good. It terms of long term maintenance, my gut feeling is to
favor the simpler solution.

G